ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -170,14 +167,17 @@ const mstp = (state: AppState): StateProps => {
   const {
     cloud: {limits},
   } = state
+  const sortOptions = state.resources.dashboards['sortOptions']
```

Comment:
Nit: it could be beneficial to use the `getByID` here:

https://github.com/influxdata/influxdb/blob/master/ui/src/resources/selectors/index.ts#L26